CMakeLists.txt

```
@@ -34,7 +34,8 @@ endif()
 
 option(BUILD_SHARED_LIBS "Build shared libraries." ON)
 option(DLAF_WITH_OPENMP "${DLAF_WITH_OPENMP_DESCRIPTION}" ${DLAF_WITH_OPENMP_DEFAULT})
-option(DLAF_WITH_MKL "Enable MKL as provider for LAPACK" OFF)
+option(DLAF_WITH_MKL "Enable oneMKL as provider for LAPACK" OFF)
```

Already there, so not a must for this PR, but:
```suggestion
option(DLAF_WITH_MKL "Enable oneMKL as provider for BLAS and LAPACK" OFF)
```
?